frontend/public/src/components/forms/new_ProfileFormFields.js

```diff
@@ -26,7 +26,20 @@ export const NAME_REGEX_FAIL_MESSAGE =
   "Name cannot start with a special character (~!@&)(+:'.?,-), and cannot contain any of (/^$#*=[]`%_;\\<>{}\"|)"
 
 export const legalAddressValidation = yup.object().shape({
-  name:     yup.string().required().label("Full Name").min(2).max(254),
+  name: yup
```

Comment:
This works but I have a thought - I was tripped up by testing this initially because I kept putting in a "real" (alphanumeric) name in Full Name and a numeric-only username in Public Username. The Public Username field is what gets stored in the `desired_edx_username` field (eventually); it's what we prefer to use for the edX username when we make the user. So - can we add this validation to that field as well?

(FWIW this was also a bit hard to test because Tutor edX allows fully numeric usernames. I was able to create an edX user with the username "654321" OK locally. This might just be a difference between what version of edX Tutor deploys versus what we have deployed - but that's a bit beyond my knowledge of edX stuff.)